### DISCUSSION TITLE
Paypal donation link ?

### DISCUSSION THREAD

**Discussion by OrpheeGT** [hrvach/deskhop#37] [General]
Hello !

I'm very hyped by your project, I ordered everything and wait to be delivered.

Do you have some paypal donation link to offer you at least a coffee ?

Thanks ;)

**Comment by hrvach** [comment]
Tell you what - let's get your device built and functional first. If you end up happy, share the project with a buddy who might find it helpful. :)

**Comment by OrpheeGT** [comment]
Already shared it :D
I have to build another one for a friend :p

**Comment by loudmx** [comment]
This an awesome project. Just read article and saw a video.   This belongs on Kickstarter. I use a windows and Linux laptop, each with their own USB C dock. My guess parts in single quantities are $20 plus cost to have PCB and case made. May $50 ea if made in quantity.  Hope you or someone take the next step.
Shared this also 
Cheers!

**Reply by loudmx** [reply]
Group I shared in immediately asked where they could buy one! I am going to research how I can have one made for me first!

**Comment by hrvach** [comment]
Thank you! Single quantity parts are actually a bit cheaper than $20, PCB is also pretty cheap (unfortunately shipping and minimum order qty always spoil things).

Elecrow is likely to soon offer bare and pre-assembled boards with or without the case on their web store to make the hardware available to more people. Hope it helps you like it helped me, I can't fit 2 keyboards on my small desk and have been struggling for ages with this problem.

**Reply by loudmx** [reply]
Please update this post with a link to Elecrow item once its there so I can share it.
Thank you.

**Comment by loudmx** [comment]
Thank you.   I will share this with my group.  

Cheers!

**Comment by loudmx** [comment]
I got a question from an friend, does it support USB 1 protocol? That
would then support Vampire.  Not sure what Vampire is though.  I only  use
Windows and Linus.   I suspect he is referring to Amiga systems as he is
huge fan of those.

Anyway, I will give this a try once I see for sale on Elecrow for sure.

**Comment by hrvach** [comment]
I believe it could work, but with non-conventional hardware like the Vampire one would have to try in order to be sure. It's a super-cool project, the Vampire - http://www.apollo-computer.com/v4standalone.php

**Comment by OrpheeGT** [comment]
I've shown it to my boss (~ 200 employees) and he asked me to keep him updated as soon as he can order some batches for some coworkers having 2 computers for different customers.

As a side note, an alternative firmware without anti-screensaver would be interesting to be corporate compliant.

And still, I request a paypal donation link :p

**Comment by ams1** [comment]
@hrvach I've been searching for ~something like this since... a loong time - tried doing something ~similar... c/c++ is not my weapon of choice... I was in the dark... and today I stumbled on this - and realised I had already starred it sometime in the past... but... didn't... 😢

This is EXCELLENT!

I'll have a look at the discussions and get involved somehow.

In the meantime, besides PayPal, could you please also enable GitHub Sponsors for donations?
